src/randoop/operation/FieldGetter.java

```diff
@@ -89,19 +86,12 @@ public ExecutionOutcome execute(Object[] statementInput, PrintStream out) {
   /**
    * appendCode adds the text for an initialization of a variable from a field to 
    * the StringBuilder.
-   * @param newVar - variable to be initialized.
    * @param inputVars - list of variables to be used (ignored).
```

Comment:
There should not be a hyphen in an `@param` line.  Could you remove them here and anywhere else they appear in the code?